eng/scripts/prepare_chat_tests.py

```diff
@@ -5,36 +5,31 @@
 
 1) Run `pip install azure.communication.administration` prior to running.
 
-2) Set the `AZURE_COMMUNICATION_CS` environment variable
+2) Run the script: `python prepare_chat_tests.py <CONNECTION_STRING>`
 
-3) Run the script.
-
-4) Copy the values to the AzureCommunicationChat scheme's testing environment variables and set `TEST_MODE` to
+3) Copy the values to the AzureCommunicationChat scheme's testing environment variables and set `TEST_MODE` to
    "record".
 """
 
-import os
+from pprint import pprint
+import sys
 from azure.communication.administration import CommunicationIdentityClient
 
-connection_string = os.environ['AZURE_COMMUNICATION_CS']
+connection_string = sys.argv[1]
 identity_client = CommunicationIdentityClient.from_connection_string(connection_string)
 
 items = { key:val for (key, val) in (x.split('=', 1) for x in connection_string.split(';')) }
 endpoint = items['endpoint']
 
 user1 = identity_client.create_user()
 user2 = identity_client.create_user()
-
 token = identity_client.issue_token(user1, scopes=["chat"]).token
 
-print('\n== AZURE_COMMUNICATION_ENDPOINT ==')
-print(endpoint)
-
-print('\n== AZURE_COMMUNICATION_USER_ID_1 ==')
-print(user1.identifier)
-
-print('\n== AZURE_COMMUNICATION_USER_ID_2 ==')
-print(user2.identifier)
+results = {
+   'endpoint': endpoint,
+   'user1': user1.identifier,
+   'user2': user2.identifier,
+   'token': token
+}

```

Comment:
I originally did this because Azure deploymentScripts required the output to be JSON... now it doesn't matter. The original might be preferable.

Reply:
Reverted this.